Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -591,6 +581,7 @@ public NodejsProjectNode(NodejsProjectPackage package)
                 }
             }
             _analyzer = analyzer;
+            TryToAcquireCurrentTypings();
```

Comment:
I know it's not part of this change, but since the operation gets called enough times (OnIdleNodeModules), we should change the internal gating of TryToAcquireTypings to check for ShouldAcquireTypings immediately rather than after it's determined the npm packages to install. Additionally, while you're at it... let's move the isNewTypingsFolder bool into the if condition in TryToAcquireTypings